docs/understanding-your-pipeline/schemas/index.md

```diff
@@ -1,37 +1,62 @@
 
-Through describing how the data should be structured as part of your schema definition you ensure clean and consistent data landing in your data warehouse or other destinations.
+Through describing how the data should be structured as part of your schema definition, you ensure clean and consistent data landing in your data warehouse or other destinations.
+
+## Building data meaning with schemas
+
+Rather than just leaving your data open to interpretation by the many different people that will consume and analyze it, you can use schemas to describe the meaning of your data. Each schema you define should clearly describe what is being collected and why — both for the schema itself, but also for each field within the schema.
+
+## Iglu
+
+**Iglu** is a set of tools for hosting and managing schemas (more specifically, JSON schemas). We use it to store all the schemas associated with events and entities.
+
+:::info Terminology
+
+We use the terms “schema repository” and “schema registry” interchangeably.
+
+:::
 
-## Building data meaning with data structures
+There is a [central Iglu repository](http://iglucentral.com/) that holds public schemas for use with Snowplow, including ones for some of the [out-of-the-box self-described events](/docs/understanding-your-pipeline/events/index.md#out-of-the-box-and-custom-events) and [out-of-the-box entities](/docs/understanding-your-pipeline/entities/index.md#out-of-the-box-entities).
 
-Rather than just leaving your data open to interpretation by the many different people that will consume and analyze it, you can use schemas to describe the meaning of your data. Each schema you define should clearly describe what is being collected and why both for the schema itself, but also for each field within the schema.
+To host schemas for your [custom self-described events](/docs/understanding-your-pipeline/events/index.md#self-describing-events) and [custom entities](/docs/understanding-your-pipeline/entities/index.md#custom-entities), you can run your own Iglu, either using [Iglu Server](/docs/pipeline-components-and-applications/iglu/iglu-repositories/iglu-server/index.md) (recommended), or a [static repository](/docs/pipeline-components-and-applications/iglu/iglu-repositories/static-repo/index.md). _(This is not required for Snowplow BDP customers, as Snowplow BDP already includes a private Iglu repository.)_
 
 ## The anatomy of a schema
 
 :::info
 If you are using BDP Cloud, you can create custom schemas using the [Data Structures Builder](/docs/understanding-tracking-design/managing-your-data-structures/builder/index.md) without worrying about how it works under the hood.
 :::
 
-Let’s take a look at an example JSON schema to talk about its constituent parts:
+Snowplow schemas are based on the JSON Schema standard ([draft 4](https://datatracker.ietf.org/doc/html/draft-fge-json-schema-validation-00)). Let’s take a look at an example schema to talk about its constituent parts:
```

Comment:
Maybe also link generally to https://json-schema.org/ as it has some more useful references like the "Learn" section etc.